### ISSUE TITLE
iceprod2 - restarted/failed task usage (Trac #836)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#830]
Keep track of restarted/failed task usage (time used, where it ran, etc).

Migrated from https://code.icecube.wisc.edu/ticket/836
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "Keep track of restarted/failed task usage (time used, where it ran, etc).", 
    "reporter": "david.schultz", 
    "cc": "ddelventhal", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "iceprod2 - restarted/failed task usage", 
    "priority": "minor", 
    "keywords": "", 
    "time": "2015-01-06T17:14:08", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150504T16:44:28: 
* david.schultz changed version from "2.0" to ""
* david.schultz changed milestone from "" to "IceProd 2.1"

**Comment by icecube-trac** [comment]
Trac update at 20160323T14:13:52: 
* david.schultz changed milestone from "IceProd 2.1" to "IceProd 2.0"
* david.schultz changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20160329T21:33:11: 
* david.schultz commented:

> In changeset 143892:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="143892"
> when a task has an error, store information about it in the task statistics (host, running time, error log tail). closes #830
> ```


* david.schultz changed status from "accepted" to "closed"
* david.schultz changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas commented:

> Milestone IceProd 2.0 deleted


* olivas changed milestone from "IceProd 2.0" to ""
